packages/cli/package.json

```diff
@@ -31,6 +31,7 @@
         "ajv-errors": "^3.0.0",
         "axios": "^1.2.0",
         "byots": "^5.0.0-dev.20221103.1.34",
```

Comment:
unrelated, do we use this `byots` package? it seems not, or at least not imported 🤔

Reply:
good question. It is a bit out of the scope of this PR. I only happen to touch CLI package.json because it wouldn't compile without explicitly set chalk as a dependency but I am sure there is a  some more cleanup to do

Reply:
Agree of out of scope but we used `byots` in an earlier iteration of the compilation of the typescript files for the cli, but now use `ts-node` instead.